docs/appendix/changelog.md

```diff
@@ -21,15 +40,42 @@ date: "2024-09-18"
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
-{{% changelog date="2024-11-05" color="added" title="TFLite model moved to module" %}}
-
-The `TFLite CPU` ML model service has moved to a [module](https://app.viam.com/module/viam/tflite_cpu).
-
-{{% /changelog %}}
-
-{{% changelog date="2024-10-28" color="added" title="Deprecate builtin pi model" %}}
-
-The built-in `pi` sensor and board models have moved to the [`raspberry-pi` module](https://github.com/viam-modules/raspberry-pi).
+{{% changelog date="2024-11-12" color="added" title="Builtin models moved to modules" %}}
+
+The following resource models have moved to modules.
+
+| Resource | Model |
+| -------- | ----- |
+| board | [`customlinux`](https://github.com/viam-modules/customlinux/) |
+| board | [`jetson`](https://github.com/viam-modules/nvidia) |
+| board | [`pca9685`](https://github.com/viam-modules/pca/tree/main) |
+| board | [`odroid`](https://github.com/viam-modules/hardkernel) |
+| board | [`ti`](https://github.com/viam-modules/texas-instruments) |
+| board | [`pi`](https://github.com/viam-modules/raspberry-pi) |
+| board | [`orange-pi`](https://github.com/viam-modules/orange-pi) |
+| board | [`upboard`](https://github.com/viam-modules/up) |
+| gripper | [`softrobotics`]() |
+| motor | [`encoded-motor`]() |
+| motor | [`gpiostepper`]() |
+| motor | [`roboclaw`]() |
```

Comment:
softrobotmics is removed, encoded-motor and gpiostepper have no changes, and roboclaw was modularized but it's in my namespace instead of the viam namespace